src/webapp/pages/app/App.tsx

```diff
@@ -53,14 +54,13 @@ export const App: React.FC<AppProps> = React.memo(function App({ api, d2, instan
                 <SnackbarProvider>
                     <LoadingProvider>
                         <HeaderBar appName="User Extended App" />
-
                         <div id="app" className="content">
                             <AppContext.Provider value={appContext}>
                                 <Router />
                             </AppContext.Provider>
                         </div>
-
```

Comment:
This is subjective but adding blank lines can improve readability for blocks that have multiline components. Another approach would be to abstract the `div#app` into another component so we will have 4 components of one line. Based on [3DBIONOTES PR #214](https://github.com/EyeSeeTea/3DBIONOTES/pull/214#discussion_r1567069654).